ts/screens/profile/PreferencesScreen.tsx

```diff
@@ -101,8 +101,7 @@ class PreferencesScreen extends React.Component<Props, State> {
       biometryTypeOrUnsupportedReason => {
         this.setState({
           isFingerprintAvailable:
```

Comment:
With this configuration, on a device without biometric hardware (`UNAVAILABLE`), I can see the menu entry.
This configuration works for me:

```suggestion
          isFingerprintAvailable:
              biometryTypeOrUnsupportedReason !== "UNAVAILABLE"
             
             ...
             
             {isFingerprintAvailable && (
            
```